<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImagesetEditorDockWidget</class>
 <widget class="QDockWidget" name="ImagesetEditorDockWidget">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>728</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>337</width>
    <height>541</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Imageset Editing</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="imagesetGroupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>172</height>
       </size>
      </property>
      <property name="title">
       <string>Imageset</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="2">
        <widget class="QLineEdit" name="nativeVertRes"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="nativeHorzRes"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="nativeResolutionLabel">
         <property name="toolTip">
          <string>Native resolution</string>
         </property>
         <property name="text">
          <string>Native res.</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QComboBox" name="autoScaled">
         <item>
          <property name="text">
           <string>false</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>min</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>max</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>true</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="name"/>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="imageLoad">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="FileLineEdit" name="image" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="imageLabel">
         <property name="text">
          <string>Underlying texture</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name</string>
         </property>
         <property name="buddy">
          <cstring>name</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="autoScaledLabel">
         <property name="text">
          <string>Auto scaled</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btnHelp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Show help overlay</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../data/Resources.qrc">
           <normaloff>:/icons/actions/help_about.png</normaloff>:/icons/actions/help_about.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="imagesGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>120</height>
       </size>
      </property>
      <property name="title">
       <string>Image definitions</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="filterBox">
            <property name="placeholderText">
             <string>Filter definitions by name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="filterResetButton">
            <property name="toolTip">
             <string>Reset the filter</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../data/Resources.qrc">
              <normaloff>:/icons/imageset_editing/filter_reset.png</normaloff>:/icons/imageset_editing/filter_reset.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="list">
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="selectionRectVisible">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="propertiesGroupBox">
      <property name="title">
       <string>Selected image definition properties</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QLabel" name="yColumnLabel">
         <property name="text">
          <string>Y</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="sizeLabel">
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="positionY">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="positionLabel">
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="autoScaledPerImageLabel">
         <property name="text">
          <string>Auto scaled</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="height"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="width"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="offsetX"/>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="offsetY"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="offsetLabel">
         <property name="text">
          <string>Offset</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="positionX"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="xColumnLabel">
         <property name="text">
          <string>X</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QComboBox" name="autoScaledPerImage">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>false</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>min</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>max</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>true</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="nativeResPerImageLabel">
         <property name="text">
          <string>Native res.</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="nativeHorzResPerImage"/>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="nativeVertResPerImage"/>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileLineEdit</class>
   <extends>QWidget</extends>
   <header>src/ui/widgets/FileLineEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../data/Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>filterResetButton</sender>
   <signal>clicked()</signal>
   <receiver>filterBox</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
